src/common/components/label-med-infobox/LabelMedInfobox.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import Infoboks from 'common/components/infoboks/Infoboks';
+import BEMHelper from 'common/util/bem';
+
+import './labelMedInfobox.less';
+
+interface LabelMedInfoboxProps {
+    title: string | React.ReactNode;
+    info?: string | React.ReactNode;
+    stil?: 'normal' | 'seksjon';
+}
+
+const LabelMedInfobox: React.StatelessComponent<LabelMedInfoboxProps> = ({ title, info, stil }) => {
+    const cls = BEMHelper('labelMedInfobox');
+    return (
+        <div className={cls.element('heading', `stil-${stil || 'normal'}`)}>
+            <h1 className={`typo-element ${cls.element('title')}`}>{title}</h1>
```

Comment:
Er det riktig med H1 i denne? H1 vil være hovedtittel i heading-hierarkiet for en innholdsseksjoner (body, section, etc), men div er ikke det. Jeg vet ikke hvilken tag som vil være riktig, men du kan i hvertfall endre til Element typen fra nav-frontend-typografi, og sette tag til strong, h2/h3 eller noe. Du må neste se på konteksten av hvordan komponenten skal brukes.

Jeg bruker headingsMap-extension i chrome for å se om heading-strukturen er fornuftig.